frontend/src/routes/FacilityBooking/ViewFacility/index.tsx

```diff
@@ -1,38 +1,20 @@
 import React, { useEffect } from 'react'
 import styled from 'styled-components'
-import TopNavBar from '../../../components/Mobile/TopNavBar'
-import bookingsIcon from '../../../assets/bookingsIcon.svg'
-import messageIcon from '../../../assets/messageIcon.svg'
-import adminIcon from '../../../assets/adminIcon.svg'
 import { useParams } from 'react-router-dom'
 import { useDispatch, useSelector } from 'react-redux'
 import { useHistory } from 'react-router-dom'
 import 'react-date-range/dist/styles.css' // main css file
 import 'react-date-range/dist/theme/default.css' // theme css file
```

Comment:
lines 6 and 7 can be removed since its not in use anymore! pls remove the imported react-date-range too

Reply:
fixed in 4c666d405